<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorWindow</class>
 <widget class="QMainWindow" name="CalculatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1418</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatorWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="lineWidth">
        <number>20</number>
       </property>
       <property name="midLineWidth">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="opaqueResize">
        <bool>true</bool>
       </property>
       <property name="handleWidth">
        <number>5</number>
       </property>
       <property name="childrenCollapsible">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="LeftWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_1">
            <attribute name="title">
             <string>Transcript 1</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QTableView" name="TranscriptView">
               <property name="autoScroll">
                <bool>true</bool>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="dragDropOverwriteMode">
                <bool>false</bool>
               </property>
               <property name="dragDropMode">
                <enum>QAbstractItemView::NoDragDrop</enum>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideRight</enum>
               </property>
               <property name="gridStyle">
                <enum>Qt::SolidLine</enum>
               </property>
               <property name="sortingEnabled">
                <bool>false</bool>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Transcript 2</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QTableView" name="TranscriptView_2"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Transcript 3</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QTableView" name="TranscriptView_3"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="RightWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="8" column="1" colspan="2">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Algorithm:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Credit</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="CreditBox">
               <property name="prefix">
                <string/>
               </property>
               <property name="suffix">
                <string/>
               </property>
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.500000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="10" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0" colspan="3">
              <widget class="QComboBox" name="comboBox">
               <property name="editable">
                <bool>false</bool>
               </property>
               <property name="currentIndex">
                <number>-1</number>
               </property>
               <property name="maxCount">
                <number>100</number>
               </property>
               <property name="minimumContentsLength">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QDoubleSpinBox" name="ScoreBox">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="value">
                <double>90.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
              </layout>
             </item>
             <item row="11" column="1">
              <widget class="QDoubleSpinBox" name="ScoreBox2">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="value">
                <double>90.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="6" column="1" colspan="2">
              <widget class="QDoubleSpinBox" name="doubleSpinBox">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>22</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="8" column="0">
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>Mark as taken</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Mark as Pending</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Change Score</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2" rowspan="2">
              <widget class="QPushButton" name="Add3">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item row="11" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Score</string>
               </property>
              </widget>
             </item>
             <item row="11" column="2">
              <widget class="QPushButton" name="Retake">
               <property name="text">
                <string>Retake</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Score</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1" colspan="2">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
               <property name="decimals">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>Change Credit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="BottomWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Initial GPA:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="IScore">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="OScore">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Overall Score:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="IGPA">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Overall GPA:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="OGPA">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Initial Score:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1418</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="actionDelete_Transcript"/>
    <addaction name="actionDelete_Course"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menu_language">
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Language</string>
     </property>
     <property name="separatorsCollapsible">
      <bool>false</bool>
     </property>
     <addaction name="action_zh"/>
     <addaction name="action_English"/>
    </widget>
    <addaction name="menu_language"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuAdd_Course">
     <property name="title">
      <string>Add Course</string>
     </property>
     <addaction name="actionAdd_a_3_Credit_95_Score_Course"/>
     <addaction name="actionAdd_a_2_Credit_95_Score_Course"/>
     <addaction name="actionAdd_a_2_Credit_89_Score_Course"/>
     <addaction name="actionAdd_a_3_Credit_89_Score_Course"/>
    </widget>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="menuAdd_Course"/>
    <addaction name="actionChange__Mode"/>
    <addaction name="actionRetake_Course"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_zh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>简体中文</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_English">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Open Transcript</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Add Algorithm</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionAdd_a_3_Credit_95_Score_Course">
   <property name="text">
    <string>Add a 3 Credit 95 Score Course</string>
   </property>
  </action>
  <action name="actionAdd_a_2_Credit_95_Score_Course">
   <property name="text">
    <string>Add a 2 Credit 95 Score Course</string>
   </property>
  </action>
  <action name="actionAdd_a_2_Credit_89_Score_Course">
   <property name="text">
    <string>Add a 2 Credit 89 Score Course</string>
   </property>
  </action>
  <action name="actionAdd_a_3_Credit_89_Score_Course">
   <property name="text">
    <string>Add a 3 Credit 89 Score Course</string>
   </property>
  </action>
  <action name="actionDelete_Transcript">
   <property name="text">
    <string>Delete Transcript</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionChange__Mode">
   <property name="text">
    <string>Change To Edit Mode</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRetake_Course">
   <property name="text">
    <string>Retake Course</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionDelete_Course">
   <property name="text">
    <string>Delete Course</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
